crates/core_arch/Cargo.toml

```diff
@@ -25,3 +25,6 @@ maintenance = { status = "experimental" }
 [dev-dependencies]
 stdarch-test = { version = "0.*", path = "../stdarch-test" }
 std_detect = { version = "0.*", path = "../std_detect" }
+
+[dependencies]
+cfg-if = "0.1.10"
```

Comment:
core_arch can't have any dependencies as it is included in libcore, which everyone else depends on, including cfg-if.

Reply:
OK. Then is there any way to use cfg_if?